src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2651,19 +2657,23 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 else
                 {
                     // Use the helper to generate a table. The table can only use a single lookup value, therefore
-                    // the two immediates index (0 to 1, in op4Reg) and rotation (0 to 3, in op5Reg) must be
-                    // combined to a single value (0 to 7)
+                    // the two immediates index and rotation must be combined to a single value
                     assert(!intrin.op4->isContainedIntOrIImmed() && !intrin.op5->isContainedIntOrIImmed());
                     emitAttr scalarSize = emitActualTypeSize(node->GetSimdBaseType());
 
-                    // Combine the two immediates into op4Reg
-                    // Shift rotation left to be out of range of index
-                    GetEmitter()->emitIns_R_R_I(INS_lsl, scalarSize, op5Reg, op5Reg, 1);
-                    // Combine the two values by ORing
+                    var_types baseType = node->GetSimdBaseType();
+
+                    const unsigned rotMask      = 0b11;
+                    const unsigned numRotBits   = genCountBits(rotMask);
```

Comment:
`numRotBits` is now unused. Could you please remove it here and in the other intrinsic implementation?